packages/react-instantsearch/src/connectors/connectHitsPerPage.js

```diff
@@ -22,17 +23,30 @@ function getCurrentRefinement(props, state) {
  * @name connectHitsPerPage
  * @kind connector
  * @propType {number} defaultRefinement - The number of items selected by default
- * @propType {{value, label}[]|number[]} items - List of hits per page options. Passing a list of numbers [n] is a shorthand for [{value: n, label: n}].
+ * @propType {{value: number, label: string||number}[]} items - List of hits per page options.
```

Comment:
Could we only allow string labels? I feel like it's unecessary to try to handle both.
Also we can say that the default label is just the value when not provided. In the docs.

What do you think?